src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksDtoToPaymentWorksVendorConversionServiceImpl.java

```diff
@@ -157,6 +192,12 @@ private boolean singleRemittanceAddressExists(PaymentWorksRequestingCompanyDTO p
                 ObjectUtils.isNotNull(pmwRequestingCompanyDTO.getRemittance_addresses().getRemittance_address()) &&
                 pmwRequestingCompanyDTO.getRemittance_addresses().getRemittance_address().size() == 1);
     }
+    
+    private boolean multipleRemittanceAddressesExist(PaymentWorksRequestingCompanyDTO pmwRequestingCompanyDTO) {
+        return (ObjectUtils.isNotNull(pmwRequestingCompanyDTO.getRemittance_addresses()) &&
+                ObjectUtils.isNotNull(pmwRequestingCompanyDTO.getRemittance_addresses().getRemittance_address()) &&
```

Comment:
With our current conventions, we typically put the "AND" operators at the beginning of the line instead of the end.

Reply:
Moved the && as requested in this method and the one above it to maintain consistency of code next to each other.